tests/i2c_master_checker.py

```diff
@@ -452,16 +454,6 @@ def handle_acked(self):
     def handle_nacked(self):
       pass
 
-    def handle_nacked_select(self):
-      # Simulate external pullup
-      self.drive_sda(1)
```

Comment:
im surpised that list line doesn't thave to make its way somewhere else?

Reply:
I think https://github.com/shuchitak/lib_i2c/blob/fix_i2c_master_checker/tests/i2c_master_checker.py#L370 should take care of that. Let me check